### PR TITLE
Added support to select timezone(#245b1ce)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -27,6 +28,28 @@ export default defineComponent({
     }
   },
   methods: {
+    async timeZoneDifferentAlert(payload: any) {
+      const alert = await alertController.create({
+                header: this.$t("Change time zone"),
```
Comment (ymaheshwari1) [inline src/App.vue:33]: Improve indentation

File: src/views/TimezoneModal.vue [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Select time zone") }}</ion-title>
+    </ion-toolbar>
+    <ion-toolbar>
+      <ion-searchbar @ionFocus="selectSearchBarText($event)" :placeholder="$t('Search time zones')"  v-model="queryString" @ionInput="findTimeZone()"></ion-searchbar>
```
Comment (ymaheshwari1) [inline src/views/TimezoneModal.vue:12]: Use self-closing tag

File: src/App.vue [hunk 3]
```diff
@@ -11,7 +11,8 @@
 import { IonApp, IonRouterOutlet, IonSplitPane } from '@ionic/vue';
 import { defineComponent } from 'vue';
 import Menu from '@/components/Menu.vue';
-import { loadingController } from '@ionic/vue';
+import { loadingController, alertController } from '@ionic/vue';
+import { useStore } from "./store";
```
Comment (ymaheshwari1) [inline src/App.vue:15]: I think we need to import `useStore` from `vuex`, as importing it from `./store` might result in some warning/error.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/TimezoneModal.vue (hunk 2 above)
Comment (disha1202) [reply src/views/TimezoneModal.vue]: Updated the code to use self closing tags.

File: src/App.vue (hunk 3 above)
Comment (disha1202) [reply src/App.vue]: Made the requested change.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**